language-design.md

```diff
@@ -59,6 +100,13 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+First and foremost, the post and its comments show us that it’s extremely important to ask for feedback during the design process.  Almost every comment provided valuable feedback, either pointing out pros and cons of the suggested names or suggesting new names entirely.  The commenters were generally in agreement that they wanted the name that felt most intuitive (7 separate commenters even used the word “intuitive” in their posts).  However, they could not seem to agree on what that name was.  Out of the 756 respondents of the [poll]( https://docs.google.com/forms/d/1pp3RY-A4MAs7b-gmqFx6bKn52_G_WLoPFkV0vueiWP4/viewanalytics?usp=form_confirm), at least 9% voted for each of the five options, meaning that for every possible name, a sizable portion of users found it to be the most intuitive.  More importantly, the most popular choice, “rgb(x) and rgba(x, a),” received only ~47% of the vote, meaning that at least half of the users would not be fully satisfied no matter what name the developers choose.  A takeaway from this, I think, is that when designing an API you may come across situations where it’s impossible to please everyone.  When each possible name has a legitimate list of pros and cons, there won’t be one right solution.  Accepting this fact and not pandering to vocal minorities is important.
+
```

Comment:
In fact, Bloch has a quote about exactly this, I think he says something like "its impossible to please everyone, so try to displease everyone equally".